src/SweetAlert/SweetAlertNotifier.php

```diff
@@ -79,7 +79,7 @@ public function message($text = '', $title = null, $icon = null)
         }
 
         if (!is_null($icon)) {
-            $this->config['icon'] = $icon;
+            $this->config['type'] = $icon;
```

Comment:
Should it be replaced from **`$this->config['type'] = $icon;`** to **`$this->config['icon'] = $icon;`**

Reply:
sweet-alert2 uses 'type' instead of 'icon'

Reply:
I don't why it is not working.
For me `$this->config['icon'] = $icon;` only working

Reply:
this change apply just laravel 6
Should it be replaced 
from     $this->config['icon'] = $icon; 
to          $this->config['type'] = $icon;